### DISCUSSION TITLE
How to use Baidu's Ernie Large Model(such as ERNIE-Bot-turbo) in SK?

### DISCUSSION THREAD

**Discussion by wangwei1237** [microsoft/semantic-kernel#4371] [Q&A] [answered]
If I want to use the Ernie Large Model in the Python version of SK, what is the recommended way now? 
Does SK already support the Ernie Large Model in the Python version?

**Comment by evchaki** [comment, marked as answer]
@wangwei1237  - you should be able to follow the patterns here - https://github.com/microsoft/semantic-kernel/tree/main/python/semantic_kernel/connectors/ai
